runtime/calamari/Cargo.toml

```diff
@@ -59,6 +59,10 @@ pallet-treasury = { git = "https://github.com/paritytech/substrate.git", default
 pallet-utility = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.37" }
 substrate-fixed = { git = "https://github.com/Manta-Network/substrate-fixed.git", default-features = false }
 
+pallet-conviction-voting = { git = "https://github.com/paritytech/substrate.git", default-features = false, branch = "polkadot-v0.9.37" }
```

Comment:
I know why you import these three pallet here, it's kind of annoying.

Reply:
yeah i just moved them from the integrations tests crate because it is excluded in some of our tests

Reply:
added to the description